packages/functional/LICENSE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2018 Solana Labs, Inc
```

Comment:
I figure you copied this from another license in this repo, but should this be 2023?

Reply:
Yeah, I copied it from I think `addresses`

#1602

Reply:
\~IANAL\~
ANALO\~
ANALOS
SNALOS
SNALOA
SOALOA
SOALNA
SOLLNA
SOLANA